docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -129,54 +144,71 @@ also signs client certificates for users connecting to Windows desktops and
 Teleport SSH servers. Teleport-protected servers and Windows desktops use these
 certificates.
 
-Once you have completed the rotation and reached the final `standby` phase,
+Before you complete the rotation and reach the final `standby` phase,
 users who have signed into Teleport must reauthenticate to receive a user
 certificate from the new CA, otherwise Teleport client commands fail.
 
-If you have registered Windows desktops with Teleport, [follow the
-guide](../../../enroll-resources/desktop-access/active-directory.mdx) to export
-the Teleport user CA so the Windows Desktop Service can authenticate to RDP
-hosts. Verify that you can connect to registered desktops throughout the
-rotation.
+If you have registered Windows desktops with Teleport,
+[follow the guide](../../../enroll-resources/desktop-access/active-directory.mdx)
+to export the Teleport user CA so the Windows Desktop Service can authenticate
+to RDP hosts.
+Verify that you can connect to registered desktops throughout the rotation.
 
 ### `db` and `db_client`
 
 The `db` and `db_client` CAs issue certificates that the Teleport Database
-Service uses to communicate with self-hosted databases. 
+Service uses to communicate with self-hosted databases.
 
 The Teleport Database Service presents a certificate signed by the `db_client`
 CA when communicating with a self-hosted database, which an admin configures to
-trust certificates issued by the CA. 
+trust certificates issued by this CA.
 
 Admins can configure self-hosted databases to present a certificate signed by
 the `db` CA, which the Database Service uses to verify that a database server is
 a genuine Teleport-protected resource. Alternatively, self-hosted databases can
 present a certificate signed by a custom CA, and admins can configure the
 Teleport Database Service to trust the CA.
 
-#### Beginning the rotation
+#### Rotating the database CAs
```

Comment:
I think it unnecessarily complicates the guide to include a complete set of instructions for the database CAs within this H4, since Step 2 purports to include instructions for all CAs. Would it be possible to spread these instructions throughout Step 2 instead?